app/assets/javascripts/snowcrash/modules/index_table.js.coffee

```diff
@@ -1,13 +1,12 @@
 class IndexTable
-  itemName: ''
-  $table: null
   project: null
   selectedColumns: []
   selectedItemsSelector: ''
   tagColumnIndex: null
 
   constructor: (@itemName) ->
-    @$table       = $('#index-table table')
+    @$jsTable     = $('.js-index-table')
+    @$table       = $('.js-items-table')
```

Comment:
this could be: `$('.js-items-table table')` ?
we are forcing a very strong binding between our code and our html